gateway/main.go

```diff
@@ -54,7 +54,7 @@ var essentialFlags = []cli.Flag{
 		Name:   "cluster",
 		Usage:  "The cluster id to start space-cloud with",
 		EnvVar: "CLUSTER_ID",
-		Value:  "default-cluster",
+		Value:  ksuid.New().String(),
```

Comment:
Is this okay? Are you able to specify a clusterid via an env variable? See if you custom id actually gets applied or not?

Reply:
ok, i'll test with env variable

Reply:
yes, it works

Reply:
Awesome